zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/handlers/IncidentHandlerTest.java

```diff
@@ -162,29 +162,78 @@ void shouldUpdateEntityFromRecord() {
     underTest.updateEntity(incidentRecord, incidentEntity);
 
     // then
-    assertThat(incidentEntity.getId()).isEqualTo(String.valueOf(recordKey));
-    assertThat(incidentEntity.getKey()).isEqualTo(123L);
-    assertThat(incidentEntity.getPartitionId()).isEqualTo(incidentRecord.getPartitionId());
-    assertThat(incidentEntity.getPosition()).isEqualTo(incidentRecord.getPosition());
-    assertThat(incidentEntity.getFlowNodeId()).isEqualTo(incidentRecordValue.getElementId());
-    assertThat(incidentEntity.getErrorMessage())
-        .isEqualTo(ExporterUtil.trimWhitespace(incidentRecordValue.getErrorMessage()));
-    assertThat(incidentEntity.getErrorType().name())
-        .isEqualTo(incidentRecordValue.getErrorType().name());
-    assertThat(incidentEntity.getBpmnProcessId()).isEqualTo(incidentRecordValue.getBpmnProcessId());
-    assertThat(incidentEntity.getProcessDefinitionKey())
-        .isEqualTo(incidentRecordValue.getProcessDefinitionKey());
-    assertThat(incidentEntity.getJobKey()).isEqualTo(incidentRecordValue.getJobKey());
-    assertThat(incidentEntity.getCreationTime())
-        .isEqualTo(
-            OffsetDateTime.ofInstant(
-                Instant.ofEpochMilli(incidentRecord.getTimestamp()), ZoneOffset.UTC));
-    assertThat(incidentEntity.getErrorMessage())
-        .isEqualTo(ExporterUtil.trimWhitespace(incidentRecordValue.getErrorMessage()));
-    assertThat(incidentEntity.getFlowNodeInstanceKey())
-        .isEqualTo(incidentRecordValue.getElementInstanceKey());
-    assertThat(incidentEntity.getErrorMessageHash())
-        .isEqualTo(incidentRecordValue.getErrorMessage().hashCode());
+    assertEntityFields(incidentEntity, recordKey, incidentRecord, incidentRecordValue);
+  }
+
+  @Test
+  void shouldUpdateEntityFromFollowUpRecord() {
```

Comment:
💭 I was not fully sure whether we really needed it. My thought was it might be that we cache some parts, and don't correctly update entity on the next read, which is why I wanted to validate it. But I guess we would need this then also on other places?